sdk/keyvault/azure-keyvault-administration/README.md

```diff
@@ -1,29 +1,231 @@
-# Azure Key Vault Administration client library for Python
+# Azure KeyVault Administration client library for .NET
+Azure Key Vault helps solve the following problems:
+- Vault administration (this library) - full backup / restore, and key-level
+role-based access control (RBAC) of vaults.
+- Cryptographic key management ([azure-keyvault-keys](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/keyvault/azure-keyvault-keys)) - create, store, and control
+access to the keys used to encrypt your data
+- Secrets management
+([azure-keyvault-secrets](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/keyvault/azure-keyvault-secrets)) -
+securely store and control access to tokens, passwords, certificates, API keys,
+and other secrets
+- Certificate management
+([azure-keyvault-certificates](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/keyvault/azure-keyvault-certificates)) -
+create, manage, and deploy public and private SSL/TLS certificates
 
 ## Getting started
+### Install packages
+Install [azure-keyvault-administration][pypi_package_administration] and
+[azure-identity][azure_identity_pypi] with [pip][pip]:
+```Bash
+pip install azure-keyvault-administration azure-identity
```

Comment:
Don't you need to also pass `--pre`?

Reply:
We never really did that with our other beta backages, @chlowell are you good with keeping it without the flag?

Reply:
Yes, `--pre` isn't required when a package has only beta versions.